README/ReleaseNotes/v624/index.md

```diff
@@ -126,6 +128,45 @@ See [Demo notebook in SWAN](https://github.com/hageboeck/rootNotebooks),
 [EPJ Web Conf. 245 (2020) 06007](https://www.epj-conferences.org/articles/epjconf/abs/2020/21/epjconf_chep2020_06007/epjconf_chep2020_06007.html),
 [arxiv:2012.02746](https://arxiv.org/abs/2012.02746).
 
+### RooBatchCompute Library
+A new library has been created with the aim of concentrating the auto-vectorizable compute functions of roofit,
+as well as all the helpers (classes, structs, helping functions) that provide this functionality.
+
+To make use of the batch evaluation feature, one must provide for their own-made PDF's the function
+```c++
+  RooSpan<double> RooMyPDF::evaluateSpan(RooBatchCompute::RunContext& evalData, const RooArgSet* normSet) const
+```
```

Comment:
```suggestion
To make use of the batch evaluation feature, one should override [`RooAbsReal::evaluateSpan()`](https://root.cern.ch/doc/master/classRooAbsReal.html#a1e5129ffbc63bfd04c01511fd354b1b8)
```c++
  RooSpan<double> RooMyPDF::evaluateSpan(RooBatchCompute::RunContext& evalData, const RooArgSet* normSet) const
\```
```

Reply:
I'm not sure that the markdown variant understands `c++`, you would need to check when the page has been generated. (Not on github, on the ROOT webpage).